src/components/forms/email-template-form.js

```diff
@@ -502,7 +513,7 @@ const EmailTemplateForm = ({ entity, match, errors, clients, preview, templateLo
             </div>
             <div className="row">
                 <div className="col-md-12 submit-buttons">
-                    <input type="button" onClick={handleSubmit}
+                    <input type="button" onClick={handleSubmit} disabled={mjmlEditor && mjmlRenderError !== null}
```

Comment:
i do think this should be move to a method, bc later we would introduce the save validation for html too